modules/appeals_api/spec/services/appeals_api/pdf_construction/generator_spec.rb

```diff
@@ -26,7 +26,7 @@
 
           it 'generates the expected pdf' do
             generated_pdf = described_class.new(notice_of_disagreement).generate
-            expected_pdf = fixture_filepath('expected_10182_minimum.pdf', version: 'v1')
+            expected_pdf = fixture_filepath('pdfs/v1/expected_10182_minimum.pdf')
```

Comment:
In this PR, I haven't yet updated the call signatures of the fixture helpers in the request specs for decision reviews or the segmented APIs - that will need to happen in later PRs to minimize the sizes of the diffs. Updates here are mostly restricted to specs for docs, models, and the pdf generator code for now.